test/executables/react16_ci_precheck_test.dart

```diff
@@ -0,0 +1,163 @@
+// Copyright 2019 Workiva Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+@TestOn('vm')
+
+import 'dart:io';
+
+import 'package:test/test.dart';
+import 'package:path/path.dart' as p;
+
+import 'package:over_react_codemod/src/creator_utils.dart';
+
+final reactPackageName = 'react';
+final overReactPackageName = 'over_react';
+
+final tansitionPubspecCreator = PubspecCreator(
+    dependencies: [DependencyCreator('react', version: '">=4.0.0 <6.0.0"')]);
+
+final versionChecksToTest = [
+  // Pubspec checks
+  DartProjectCreatorTestConfig(
+    testName:
+        'does not run the codemod when project does not have a pubspec.yaml or if it is no parsable',
```

Comment:
This message should be updated, since this pre-check command sets the exit code and doesn't actually run codemods.

Same goes for other discriptions in this file